### PR TITLE
Implement ICA (image-cloze-association)

### PR DESCRIPTION
**Pull request by sergiulucaci**
If you notice anything that can be improved please shoot.

### CODE REVIEW

File: packages/image-cloze-association/src/root.jsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+    this.state = {
+      answers: [],
+      draggingElement: { id: '', value: '' },
+      possibleResponses: possibleResponses.map(item => ({ id: generateId(), value: item })),
+      responseContainers: responseContainers.map((item, index) => ({ id: generateId(), index, ...item }))
+    };
+  }
+
+  handleOnDragBegin = (draggingElement) => {
```
Comment (edeustace) [inline packages/image-cloze-association/src/root.jsx]: `beginDrag` so .... `onDragBegin={this.beginDrag}`

File: packages/image-cloze-association/src/utils-correctness.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const getAllCorrectAnswers = (answers, validResponses) =>
+  answers.map(answer => ({
+    ...answer,
+    isCorrect: validResponses[answer.containerIndex].includes(answer.value)
+  }));
+
+const getUniqueCorrectAnswers = (answers, validResponses) => {
```
Comment (edeustace) [inline packages/image-cloze-association/src/utils-correctness.js]: test?

File: packages/image-cloze-association/src/interactive-section.jsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  }
+
+  render() {
+    const { children } = this.props;
+    const classname = this.getClassname();
+    const Icon = this.getIcon();
+
+    return (
+      <div className={classname}>
+        {Icon}
```
[the remark this answers is not in the document]
Comment (sergiulucaci) [reply packages/image-cloze-association/src/interactive-section.jsx]: Can you please describe a bit where should I point for the theme changes? (not sure if its something related to this package, or pie-ui or higher)

[the remark this answers is not in the document]
Comment (edeustace) [reply packages/image-cloze-association/src/interactive-section.jsx]: it's ok you can leave out theme changes for now, it's something we should get to soon, but can be addressed elsewhere.

**Pull request merged**